### ISSUE TITLE
Be able to use .includeFormattedValuesFetch when using .useFetchXml

### ISSUE THREAD

**Issue by ferodom** [delegateas/XrmDefinitelyTyped#51]
Maybe I am missing something but I am not able to use .includeFormattedValuesFetch() when my query is based on fetchxml

**Comment by mktange** [comment]
You are correct. It is currently not possible to include formatted values when using FetchXml with XrmQuery.

**Comment by ferodom** [comment]
is that something is coming? It is not a restriction with the WebAPI correct?

**Comment by TomMalow** [comment]
No it is not a restriction in the WepAPI but simply our API which does not currently support it. We will look into adding .includeFormattedValuesFetch when using .useFetchXml at some point. Can't say when.

Furthermore, we are advising not to use FetchXML if possible.
